dynamodb-stream-reader/src/main/java/no/unit/nva/dynamodb/DynamoDBStreamHandler.java

```diff
@@ -5,8 +5,8 @@
 import com.amazonaws.services.lambda.runtime.events.DynamodbEvent;
 import com.amazonaws.services.lambda.runtime.events.models.dynamodb.AttributeValue;
 import no.unit.nva.elasticsearch.Constants;
-import no.unit.nva.elasticsearch.ElasticSearchIndexDocument;
 import no.unit.nva.elasticsearch.ElasticSearchRestClient;
+import no.unit.nva.elasticsearch.IndexDocument;
```

Comment:
Lines 31 and 32 have fields that are not in use.

Reply:
This is addressed in another previous PR